### PR TITLE
Update avs-dashboard-onboarding.md

### PR DESCRIPTION
**Pull request by samlaf**
PR following this slack thread: https://eigenlabs.slack.com/archives/C060ARU8AF3/p1713493878709149

### CODE REVIEW

File: docs/eigenlayer/avs-guides/avs-dashboard-onboarding.md [hunk 1]
```diff
@@ -52,3 +52,6 @@ The metadataURI should follow the format outlined in this [example](https://hole
 
 ## Holesky Dashboard onboarding
 Once you've gone through the above steps, fill out [this form](https://forms.gle/8BJSntA3eYUnZZgs8) so the metadata you submitted for your AVS can be reflected on the Holesky dashboard.
+
+## Mainnet Dashboard onboarding
+Contact us.
```
Comment (samlaf) [inline docs/eigenlayer/avs-guides/avs-dashboard-onboarding.md:57]: @NimaVaziri is this true? Or are we permissionless now? what about holesky?

File: docs/eigenlayer/avs-guides/avs-dashboard-onboarding.md [hunk 2]
```diff
@@ -1,27 +1,27 @@
...
+// Below 3 functions are just proxies to the same-named functions in the AVSDirectory
 function registerOperatorToAVS(address operator, Signature memory signature);
 
 function deregisterOperatorFromAVS(address operator);
 
 function updateAVSMetadataURI(string calldata metadataURI);
 	
-	// Implement in ServiceManager
-	function getOperatorRestakedStrategies(address operator) returns (address[] memory)
+// Below 2 functions are needed for the frontend and payments MVP
```
Comment (samlaf) [inline docs/eigenlayer/avs-guides/avs-dashboard-onboarding.md:21]: Added mention of payments MVP here. Not sure if we want to wait until it's actually live before having it on the official docs. But we also have to remember to add it as soon as we launch payments if we don't have it now.

Comment (wesfloyd) [reply docs/eigenlayer/avs-guides/avs-dashboard-onboarding.md:21]: @samlaf yes - if possible, I'd love for us to hold off on that content until the MVP is live. Thank you

File: docs/eigenlayer/avs-guides/avs-dashboard-onboarding.md (hunk 1 above)
Comment (wesfloyd) [inline docs/eigenlayer/avs-guides/avs-dashboard-onboarding.md:57]:
```suggestion
Prior to planning your Mainnet onboarding, please test first in Testnet and contact our team via the form [here](https://docs.eigenlayer.xyz/eigenlayer/avs-guides/avs-dashboard-onboarding#holesky-dashboard-onboarding).
```

**Review by wesfloyd — CHANGES_REQUESTED** [review]

File: docs/eigenlayer/avs-guides/avs-dashboard-onboarding.md (hunk 2 above)
Comment (samlaf) [reply docs/eigenlayer/avs-guides/avs-dashboard-onboarding.md]: [67f14c7](https://github.com/Layr-Labs/eigenlayer-docs/pull/208/commits/67f14c7ac2537ac5fe228db009bef11bc4995c8c)

**Review by wesfloyd — APPROVED** [review]

**Pull request merged**